communities.md

```diff
@@ -100,3 +100,83 @@ Example: not available
 The supported parameters are:
 * **(mandatory)** parent, the UUID of the parent community
 * page, size [see pagination](README.md#Pagination)
+
+## Creating communities
+
+### Creating top level community
+
+**POST /api/core/communities**
+
+To create a top level community, perform as post with the JSON below to the communities endpoint when logged in as admin.
+
+```
+{
+    "name": "test creation",
+    "metadata": [
+        {
+            "key": "dc.title",
+            "value": "test creation",
+            "language": null
+        }
+    ]
+}
+```
+
+### Creating subCommunity
+
+**POST /api/core/communities?parentCommunity=<:communityUUID>**
```

Comment:
I suggest to use parent as parameter name as above

Reply:
The parameter has been renamed. There's no potential ambiguity in the name parent since no different type of parent can be specified